Sources_v3/StreamChat/APIClient/Endpoints/Payloads/CurrentUserPayloads.swift

```diff
@@ -61,8 +61,46 @@ class CurrentUserPayload<ExtraData: UserExtraData>: UserPayload<ExtraData> {
 }
 
 /// An object describing the outgoing user JSON payload.
-class CurrentUserRequestBody<ExtraData: UserExtraData>: Encodable {
-    // TODO: Add more fields while working on CIS-235
+struct CurrentUserUpdateRequestBody<ExtraData: UserExtraData>: Encodable {
```

Comment:
This can be just `UserUpdateRequestBody`

Reply:
Same as above, should this move to `UserPayloads` ?

Reply:
@nuno-vieira I'd say so 🤷‍♂️